www/source/partials/docs/_using-builder-origin-keys.html.md.erb

```diff
@@ -0,0 +1,129 @@
+## <a name="origin-keys" id="origin keys" data-magellan-target="origin-keys">Using Origin Keys</a>
+
+## About origin keys
+
+Creating an origin automatically generates an origin key pair. 
+
+Origin key cryptography is asymmetric: it has a public origin key that you can distribute freely, and a private origin key that you should distribute only to members of the origin.
+
+Habitat Builder and Habitat Depot require access to at least the public origin key to validate packages in the origin. 
+
+Habitat uses the public origin key to verify the integrity of a downloaded Habitat artifact before installing it.
+
+By default, Habitat only installs packages for which ir has the public origin key.
+You can provide the public origin key to Habitat by pointing it to a depot that containing it, or by putting the public origin key on disk outside of Habitat in a location that Habitat knows to look for it, such as `~/.hab/cache/keys` on Linux systems or `C:\hab\cache\keys` on Windows systems.
```

Comment:
```suggestion
You can provide the public origin key to Habitat by pointing it to a depot containing it, or by putting the public origin key on disk outside of Habitat in a location that Habitat knows to look for it, such as `~/.hab/cache/keys` on Linux systems or `C:\hab\cache\keys` on Windows systems.
```

`~/.hab/cache/keys` and `C:\hab\cache\keys` aren't "outside of Habitat", and we don't want to encourage users to manually do anything with these directories, since their behavior may change in the future. All documentation should refer users to the appropriate `hab` commands for key management. It's probably best to replace this section with a reference to the other section which describes downloading/importing public origin keys in more detail.

Reply:
Agree!

Reply:
I suggest we delete the line.